api/openapi-spec/swagger.json

```diff
@@ -786,7 +799,11 @@
           "description": "Rolling update config params. Present only if type = \"RollingUpdate\"."
         },
         "type": {
-          "description": "Type of daemon set update. Can be \"RollingUpdate\" or \"OnDelete\". Default is RollingUpdate.",
+          "description": "Type of daemon set update. Can be \"RollingUpdate\" or \"OnDelete\". Default is RollingUpdate.\n\nPossible enum values:\n - `OnDelete`: Replace the old daemons only when it's killed\n - `RollingUpdate`: Replace the old daemons by new ones using rolling update i.e replace them on each node one after the other.",
```

Comment:
That sort of comment should eventually be fixed, since the enum is already explained in plain english. That's obviously fine for now.